sycl/test-e2e/SYCLBIN/test-plan.md

```diff
@@ -0,0 +1,156 @@
+# Test plan for `sycl_ext_oneapi_syclbin`
+
+Spec: <https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_syclbin.asciidoc>  
+
+Design docs:
+
+- <https://github.com/intel/llvm/blob/sycl/sycl/doc/design/SYCLBINDesign.md>
+- <https://github.com/intel/llvm/blob/sycl/sycl/doc/design/PropertySets.md>
+
+Tests:
```

Comment:
It also feels like some bigger e2e scenarios are missing here. I would work with @steffenlarsen to ensure that key scenarios from the main customer for this feature are also incorporated into the test plan - and they could give us inspiration for checking some more cases or scenarios.

For example, what if `syclbin` is loaded by a shared library and then transferred to an application to be used?